modules/dashboard/widgets/xy-promql/main.tf

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2025 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+output "widget" {
+  value       = {
+    title = var.title
+    xyChart = {
+      chartOptions = { mode = "COLOR" }
+      dataSets = [{
+        plotType   = var.plot_type
+        targetAxis = "Y1"
+        timeSeriesQuery = {
+          prometheusQuery = var.promql_query
+        }
+      }]
+      thresholds = [
+        for threshold in var.thresholds : {
+          value      = threshold
+          targetAxis = "Y1"
+        }
+      ],
```

Comment:
should we keep this as a list of threshold objects to keep it the same as the other widgets?

https://github.com/chainguard-dev/terraform-infra-common/blob/main/modules/dashboard/widgets/xy-ratio/main.tf#L50